wrappers/src/app_cs.cpp

```diff
@@ -263,6 +274,15 @@ extern "C" {
         });
     }
 
+    REALM_EXPORT void shared_app_delete_user(SharedApp& app, SharedSyncUser& user, void* tcs_ptr, NativeException::Marshallable& ex)
+    {
+        return handle_errors(ex, [&]() {
+            app->delete_user(user, [tcs_ptr](util::Optional<AppError> err) {
```

Comment:
Can this use `get_callback_handler`?

Reply:
I read `get_callback_handler` too quickly 🤦🏻 .